subsys/bluetooth/mesh/Kconfig

```diff
@@ -46,6 +46,21 @@ config BT_MESH_USES_MBEDTLS_PSA
 
 endchoice
 
+if BT_MESH_USES_MBEDTLS_PSA
+
+config BT_MESH_PSA_KEY_ID_USER_MIN_SHIFT
```

Comment:
I find it a bit tricky to understand exactly what this parameter is from the description. Do you think it is possible to write it in a more comprehendable way, or is it just that complex?

Reply:
I guess we have a bit different point of view in this. I've already spent significant time to work with PSA specification and changes related to PSA adaptation. Description seems quite clear for me. If you suggest clarifications those help understanding I'll try to integrate them.